gnucash/register/register-gnome/combocell-gnome.c

```diff
@@ -87,28 +87,43 @@ static void gnc_combo_cell_destroy (BasicCell *bcell);
 
 static GOnce auto_pop_init_once = G_ONCE_INIT;
 static gboolean auto_pop_combos = FALSE;
+static gboolean type_ahead_search = FALSE;
 
 
+// Two Callbacks called when the user changes preferences
 static void
 gnc_combo_cell_set_autopop (gpointer prefs, gchar *pref, gpointer user_data)
 {
     auto_pop_combos = gnc_prefs_get_bool (GNC_PREFS_GROUP_GENERAL_REGISTER,
                                           GNC_PREF_AUTO_RAISE_LISTS);
 }
 
+static void
+gnc_combo_cell_set_type_ahead_search (gpointer prefs, gchar *pref, gpointer user_data)
+{
+    type_ahead_search = gnc_prefs_get_bool (GNC_PREFS_GROUP_GENERAL_REGISTER,
+                                          GNC_PREF_TYPE_AHEAD_SEARCH);
+}
+
 static gpointer
 gnc_combo_cell_autopop_init (gpointer unused)
 {
     gulong id;
     auto_pop_combos = gnc_prefs_get_bool (GNC_PREFS_GROUP_GENERAL_REGISTER,
                                           GNC_PREF_AUTO_RAISE_LISTS);
-
-    id = gnc_prefs_register_cb (GNC_PREFS_GROUP_GENERAL_REGISTER,
```

Comment:
Why have you changed this, with your changes there is now no way to remove the auto raise callback, you will probably have to do some thing similar to remove your callback unless I have missed something.

Reply:
Crap. i think that's a mistake. I don't know how it happened. Sorry about that. I'll revert that change.

Reply:
Looks like the indentation got messed up when you put it back.